stochproc/timeseries/models/self_exciting_process.py

```diff
@@ -76,7 +76,7 @@ def kernel(self, x: TimeseriesState, alpha, xi, eta) -> Distribution:
         diffusion = eta * dl_t.abs()
         lambda_t = (lambda_s + deterministic + diffusion).clip(min=0.0)
 
-        combined = torch.stack((lambda_t, dl_t), dim=-1)
+        combined = torch.stack((lambda_t, dl_t, lambda_s), dim=-1)
```

Comment:
You'll also need to add the "initial sample" for lamda_s in the "initial_kernel", guess it should be 0?